Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8/python-39@sha256:496f9eee1837022e749a5f2d5c9ae91720200e03c38528894aee4eb6b3bb78b5
+
+USER root
+
+RUN INSTALL_PKGS="flask ansible_runner ansible-pylibssh jsonschema GitPython requests" && \
+    pip install $INSTALL_PKGS
+
```

Comment:
More broadly, you probably want to place your Python requirements into a `requirements.txt` file to make it easier to install into a development environment. Then your Dockerfile would look something like:

```
COPY requirements.txt ./
RUN pip install -r requirements.txt
COPY app.py ./
```

By copying `requirements.txt` and performing the `pip install` before copying over `app.py`, you take advantage of Docker's layer cache so that you won't need to re-run the `pip install` unless you change `requirements.txt`